src/Sulu/Bundle/AdminBundle/Resources/js/components/Modal/ModalBox.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+import React from 'react';
+import keydown from 'react-keydown';
+import Icon from '../Icon';
+import type {ModalProps} from './types';
+import Actions from './Actions';
+import modalBoxStyles from './modalBox.scss';
+
+type Props = ModalProps & {
+    onRequestClose: () => void,
+}
+
+const CLOSE_ICON = 'times';
+
+export default class ModalBox extends React.PureComponent<Props> {
```

Comment:
`OverlayBox`? `Box` is strange somehow, because it doesn't transport a real message for me... Maybe there is a better alternative.